### ISSUE TITLE
Env file comments cannot be decrypted using published Sops binary

### ISSUE THREAD

**Issue by ncjones** [mozilla/sops#454]
I have a secrets.env file which contains comment lines. I can encrypt and decrypt this file with Sops on my Ubuntu host:

    ⇒  sops -d env/test.secrets.env 
    # my comment
    MY_KEY=bc738e905caf11e9a759af4c917823f8

But I cannot decrypt the file when I run sops inside a new Debian-based docker container:

    ⇒  docker run -ti --rm \
      -v ~/.config/gcloud:/root/.config/gcloud \
      -v $(pwd):/work \
      --workdir=/work \
      sops sops -d env/test.secrets.env

> Error decrypting tree: Error walking tree: Could not decrypt value: Input string ,iv:pOEodIt8I979Dlz5Pm2NkvgT6EXMf+1mUP7T9oT0IQ8=,tag:dV89y1szk9ovLEzHJ0Sbvg==,type:comment] does not match sops' data format

The Dockerfile is:

```Dockerfile
FROM buildpack-deps:stretch
ENV SOPS_VERSION=3.2.0
ENV SOPS_BINARY=/usr/bin/sops
RUN curl -sL "https://github.com/mozilla/sops/releases/download/${SOPS_VERSION}/sops-${SOPS_VERSION}.linux" > "${SOPS_BINARY}"
RUN chmod +x "${SOPS_BINARY}"
```

Using the mozilla/sops Docker image does work but it's based on an unsupported version of the golang base image so it's hard to tell what's different about it.

Any idea what is going on here?

**Comment by ncjones** [comment]
Oh I did not realize my local sops version was installed via `go get`. When I switched to the published 3.2.0 binary I get the same behavior outside of a container.

**Comment by ncjones** [comment]
Duplicate of #451
